multimodal/src/autogluon/multimodal/models/fusion.py

```diff
@@ -355,26 +367,89 @@ def __init__(
             projection=False,
             additive_attention=additive_attention,
             share_qv_weights=share_qv_weights,
+            row_attention=row_attention,
+            row_attention_layer=row_attention_layer,
+            global_token=global_token,
         )
 
-        self.head = FT_Transformer.Head(
-            d_in=in_features,
-            d_out=num_classes,
-            bias=True,
-            activation=head_activation,
-            normalization=head_normalization,
+        self.remainder_transformer = FT_Transformer(
```

Comment:
What is this `remainder_transformer ` used for? Can we merge it with `fusion_transformer`? A fusion model usually have three parts, `model` (the single models), `fusion_xxx` (fusion), and `head` (final output). Having an additional `remainder_transformer` doesn't follow this simple tradition and bring corner cases when analyzing the model structure.